### ISSUE TITLE
Difference between recognition pretrained model and default english recognition model?

### ISSUE THREAD

**Issue by cyberjj999** [PaddlePaddle/PaddleOCR#3951]
Hi! I am trying to compare the differences between the default english recognition model (PaddleOCR release v2.1)
``` 
'en': {
            'url':
            'https://paddleocr.bj.bcebos.com/dygraph_v2.0/multilingual/en_number_mobile_v2.0_rec_infer.tar',
            'dict_path': './ppocr/utils/en_dict.txt'
        },
```
compared to the pretrained (MobileNetV3) model mentioned in the recognition training.
```
cd PaddleOCR/
# Download the pre-trained model of MobileNetV3
wget -P ./pretrain_models/ https://paddleocr.bj.bcebos.com/dygraph_v2.0/en/rec_mv3_none_bilstm_ctc_v2.0_train.tar
# Decompress model parameters
cd pretrain_models
tar -xf rec_mv3_none_bilstm_ctc_v2.0_train.tar && rm -rf rec_mv3_none_bilstm_ctc_v2.0_train.tar
```
However, when I did my custom evaluation, the evaluation results are surprisingly similar. May I know if they are actually similar models?
Otherwise, may I know the general differences between them?

Thanks in advance!

**Comment by MissPenguin** [comment]
The training set is different. How many images in your evaluation set? It's possible that these two models achieve similar accuracy on some images.

**Comment by cyberjj999** [comment]
@MissPenguin Hi. There's about 300+ images in my evaluation set. Other than the training set difference, may I know if there are other differences that is good for me to note?
Thanks in advance!
